backends/firrtl/firrtl.cc

```diff
@@ -392,7 +392,37 @@ struct FirrtlWorker
 		return result;
 	}
 
-	void run()
+	void emit_extmodule()
+	{
+		std::string moduleFileinfo = getFileinfo(module);
+		f << stringf("  extmodule %s: %s\n", make_id(module->name), moduleFileinfo.c_str());
+		vector<string> port_decls;
+
+		for (auto wire : module->wires())
+		{
```

Comment:
There is an opening brace on a new line here but not for the `for` on line 418.  Personally, I prefer to keep the opening brace on the same line, but either way, it should be consistent.

Reply:
Nice catch. My muscle memory put the brace on the same line as I also prefer it, but (most of) the rest of this source file had them on the following line, so I stuck with that convention.